pkg/repository/v1/olap.go

```diff
@@ -303,6 +308,10 @@ func (o *OLAPRepositoryImpl) UpdateTablePartitions(ctx context.Context) error {
 	return nil
 }
 
+func (o *OLAPRepositoryImpl) SetReadReplicaPool(pool *pgxpool.Pool) {
+	o.readPool = pool
```

Comment:
so the idea here is by default it'll use the leader unless a replica is provided?

Reply:
yep, exactly